Mage.Client/src/main/java/org/mage/plugins/card/CardPluginImpl.java

```diff
@@ -186,7 +186,7 @@ private Row createStacks(Map<UUID, MageCard> cards, Row workingRow, RowType rowT
                         ? firstPanelPerm.getOriginal().getRules()
                         : new ArrayList<>();
                 // Check the names are equal and are creatures with the same summoning sickness
-                if (firstPanelPerm.getOriginal().getName().equals(perm.getOriginal().getName())
+                if (firstPanelPerm.getOriginal().isToken() && firstPanelPerm.getOriginal().getName().equals(perm.getOriginal().getName())
```

Comment:
Looks like it must be checked same way as other fields like


```
firstPanelPerm.getOriginal().isToken() == perm.getOriginal().isToken()
firstPanelPerm.getOriginalPermanent().hasSummoningSickness() == perm. getOriginalPermanent().hasSummoningSickness()
```

Reply:
Correct, lands weren't stacking, fixed

![image](https://github.com/magefree/mage/assets/19590931/a2b542d1-8514-4949-bdc3-b87f5b6db9e4)